plugins/modules/system/syspatch.py

```diff
@@ -85,7 +83,7 @@
 def run_module():
     # define available arguments/parameters a user can pass to the module
     module_args = dict(
-        apply=dict(type='bool'),
+        apply=dict(type='bool', default=True),
```

Comment:
This breaks backwards compatibility.

Reply:
That's okay. If running without any arguments defined right now results in a failure:

```
File \"/tmp/ansible_syspatch_payload_z1wmzxqc/ansible_syspatch_payload.zip/ansible/modules/system/syspatch.py\", line 169, in <module>\n  File \"/tmp/ansible_syspatch_payload_z1wmzxqc/ansible_syspatch_payload.zip/ansible/modules/system/syspatch.py\", line 165, in main\n  File \"/tmp/ansible_syspatch_payload_z1wmzxqc/ansible_syspatch_payload.zip/ansible/modules/system/syspatch.py\", line 95, in run_module\n  File \"/tmp/ansible_syspatch_payload_z1wmzxqc/ansible_syspatch_payload.zip/ansible/modules/system/syspatch.py\", line 119, in syspatch_run\nUnboundLocalError: local variable 'check_flag' referenced before assignment\n
```

This won't break backward compatibility when `revert` is used, since it will take precedence over `apply`.

Reply:
Then I'm wondering why the `apply` option is there at all.

Reply:
Yeah we could easily remove it, but that would be breaking backwards compatibility. We could easily deprecate it for a few releases and then remove it entirely.

I'll get the changelog fragment added.